calculator/main.py

```diff
@@ -37,39 +51,38 @@ def add_new_operation(calc, operation):
     :param operation: Dict with the single operation to be added.
                       ie: {'add': add_function}
     """
-    pass
+    try:
```

Comment:
Thanks! :) Why do the work when you can have something else do the work for you? ;)